lib/fs.js

```diff
@@ -1606,21 +1606,20 @@ function ReadStream(path, options) {
     return new ReadStream(path, options);
 
   // a little bit bigger buffer and water marks by default
-  options = util._extend({
-    highWaterMark: 64 * 1024
-  }, options || {});
+  options = Object.create(options || {});
```

Comment:
added this because i realized that without it, al my `.hasOwnProperty()` stuff is futile